index.ts

```diff
@@ -1,4 +1,20 @@
-import { Configs, KlasaClient } from './imports';
+import * as dotenv from 'dotenv';
+import { KlasaClient } from './imports';
+
+dotenv.config();
+let path;
+switch (process.env.NODE_ENV) {
+  case 'test':
+    path = `${__dirname}/../../.env.test`;
+    break;
+  case 'production':
+    path = `${__dirname}/../../.env.production`;
+    break;
+  default:
+    path = `${__dirname}/../../.env.development`;
+}
+dotenv.config({ path });
```

Comment:
Why 2 configs? 1 above empty then another here?

Reply:
`dotenv.config();` is basically setting up all the cases, `dotenv.config({ path });` is parsing the path's in the cases, if I don't set this config up, it doesn't allow process.env.token